### PR TITLE
added med cat toggle on patrol

### PR DESCRIPTION
**Pull request by sarhecker**
Toggles if med cats are included in +1 +3 +6

### CODE REVIEW

File: scripts/game_structure/windows.py [hunk 1]
```diff
@@ -621,6 +621,83 @@ def process_event(self, event):
                 game.all_screens['profile screen'].screen_switches()
                 self.kill()
 
+class ExileCat(UIWindow):
```
Comment (keyraven) [inline scripts/game_structure/windows.py:624]: Was this intended to be added in the PR? It is not mentioned in the title or summary. It is also causing the pylint test fail.

Comment (sarhecker) [reply scripts/game_structure/windows.py:624]: Yeah I'm just not sure how to create separate PRs ;-;

Comment (keyraven) [reply scripts/game_structure/windows.py:624]: I'd be happy to help! You need to create a new branch on your fork, and then you can put whatever changes you want on that branch, then PR from that branch.  This branch has a lot of stuff on it, so I suggest creating a new branch and copy-and-pasting all the changed you want for a single PR. Then creating a new PR from that fork.

Comment (keyraven) [reply scripts/game_structure/windows.py:624]: If you want a more step-by-step guide, message me on discord!

**Pull request closed without merging**